### PR TITLE
added my personal resource garden

### PR DESCRIPTION
**Pull request by Raehale**
Added my resources based on the basic layout provided

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,6 +13,121 @@ <h1>A digital garden, open-sourced by the community</h1>
...
     <main>
 
+        <section>
+            <h2>Rae's favorite resources</h2>
+            <div class="container">
+                <ul class="inner">
+                    <li>
+                        <a href="https://www.chingu.io/" target="_blank" class="">
+                            Chingu - Collaborate with developers around the world
+                        </a>
```
Comment (michaella23) [inline index.html:23]: closing `li` tag after this

**Review by michaella23 — CHANGES_REQUESTED** [review]
Wrap each anchor in a list item

**Review by michaella23 — APPROVED** [review]
Thanks for making the changes!

**Comment by egpennington** [comment]
Wow!  Awesome resources

**Pull request merged**